Sources/BoxSDK.swift

```diff
@@ -181,10 +182,12 @@ public class BoxSDK {
         public func getOAuth2Client(
             tokenInfo: TokenInfo? = nil,
             tokenStore: TokenStore? = nil,
+            callbackURL: String? = nil,
```

Comment:
I initially picked this approach of adding the callbackURL to `getOAuth2Client`, because the callbackURL only applies to the OAuth 2 auth type. But I'm thinking it may be cleaner to instead add an optional callbackURL parameter to the initializer, so users could instantiate the sdk as `BoxSDK(clientId: "", clientSecret: "", callbackURL: "")`. I'd then store it in BoxSDKConfiguration so it would be available as `configuration.customCallbackURL`.

Thoughts & suggestions much appreciated!

Reply:
I ended up pushing a new commit with the implementation I mentioned above. Take a look at the first commit if you'd like to compare the approaches.

Reply:
I like the approach of the second commit because callback url is an app setting like client id and secret, so it makes sense to set all of them together.